dependencies/spirv/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+include(FetchContent)
```

Comment:
Would it please be possible to make use of the packaged version of the headers instead of always fetching them?
I would imagine that would require having a `cmake/modules/FindSPIRVHeaders.cmake`,
which would look for them in a configurable set of directories.
https://packages.debian.org/sid/all/spirv-headers/filelist

Reply:
@LebedevRI Hi! Yes, that does seem to be a better solution for integrating into existing build environments.  I'll add that to my list!  Thanks for the feedback!